Ko translation

yonguilee opened the pull request.

yonguilee commented:
@slhenry  hi, Shawn :)  It's been a while. 
I have a problem.
I made a MR for checking the view of Korean translation, but Language Shift doesn't work. (in deploy/netlify link)

When I click Korean, the view still English. Can you check it for me ?

Thank you : -)!

slhenry commented:
Hi @yonguilee :) Great that you are doing more translations.

Just one "ko" was missing :-) https://github.com/yonguilee/wai-mobile/commit/5574d44a09f77b1d6d5051f103ebb75f5020885b#diff-1b0506aabfbb1f6392720f05fed391a6

As always, let me know if you have suggestions for making the instructions more clear, e.g., [front matter](https://www.w3.org/wiki/WAI/Translation_Instructions#2._Instructions_Here_for_Front-matter)

Thanks again for your translations!

yonguilee commented:
hi, @slhenry ! It's been a while again :) Sorry for late..

 I updated my translation and fixed some error reviewed from my coworker Jun Ho Lee.

Reviewer info : Jun Ho Lee (gomjunho@gmail.com)

check it please :) and Happy Christmas!🎄🎄🎄
